src/mscorlib/src/System/Environment.cs

```diff
@@ -591,6 +596,8 @@ public static String[] GetCommandLineArgs()
              */
             if(s_CommandLineArgs != null)
                 return (string[])s_CommandLineArgs.Clone();
+#else
+            new EnvironmentPermission(EnvironmentPermissionAccess.Read, "Path").Demand();
```

Comment:
As a general rule, we're removing CAS from the .NET Core codepath, as long as we're confident that it doesn't have necessary sideffects (like IO exception) -- is that right?

And we can't simply remove these calls because coreclr isn't just used in .NET Core ?